ts/store/reducers/content.ts

```diff
@@ -116,7 +119,28 @@ export default function content(
         servicesMetadata: { ...initialContentState.servicesMetadata }
       };
 
+    case getType(removeServiceTuples): {
+      const serviceTuples = action.payload;
+      const newServicesMetadataByIdState = {
+        ...(state as any).servicesMetadata.byId
+      };
+      serviceTuples.forEach(
```

Comment:
Since delete is a very slow operation you could try to change approach and also avoid object mutation

Could you check if this approach can do the same? (I did't test it)

```typescript
const servicesMetadataUpdated = Object.keys(
        newServicesMetadataByIdState
      ).reduce(
        (object, key) => {
          const hasValue = serviceTuples.find(t => t.e1 === key);
          if (hasValue) {
            return { ...object, key: newServicesMetadataByIdState[key] };
          }
          return object;
        },
        {} as ServiceMetadataById
      );
```

Reply:
I compared the 2 approaches forcing the removal of all the metadata items and the current approach seems to be less expensive. Here the time requested:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/38431762/64242083-3277aa00-cf05-11e9-844c-f1aff34faf3c.png">

Reply:
@valentinacala ok you could keep your approach. Anyway if you could add a comment explaining what's happening in that case